src/test/java/no/unit/nva/doi/transformer/MainHandlerTest.java

```diff
@@ -111,18 +110,18 @@ public  void testInternalServerErrorResponse() throws IOException {
     }
 
     private Context getMockContext() {
-        Context context = mock(Context.class);
-        CognitoIdentity cognitoIdentity = mock(CognitoIdentity.class);
-        when(context.getIdentity()).thenReturn(cognitoIdentity);
-        when(cognitoIdentity.getIdentityPoolId()).thenReturn("junit");
-        return context;
+        return mock(Context.class);
     }
 
     private InputStream inputStream() throws IOException {
         Map<String, Object> event = new HashMap<>();
-        String body = new String(Files.readAllBytes(Paths.get("src/test/resources/datacite_response.json")));
+        String body = new String(Files.readAllBytes(Paths.get("src/test/resources/datacite_response2.json")));
```

Comment:
"datacite_response2.json" as constant and use YourClass.class.getResourceAsStream(filename), or something like that?